uint32 ped_id
ford_msgs/Pose2DStamped[] traj